From 47ba3523afa99115eaeb2060399b7f3250e19a8b Mon Sep 17 00:00:00 2001
From: Andy Lavr <andy.lavr@gmail.com>
Date: Tue, 13 Nov 2018 16:27:47 +0200
Subject: [PATCH] bfq: Increase BLKCG_MAX_POLS

Upstream 4.19 added another blk-cgroup policy, for a total of 4. They
also raised the policy limit to 5, just as the BFQ patchset for 4.18
has.

BFQ adds 2 policies, which now makes a total of 6, above the limit.
Increase the limit to 7 to compensate.
---
 include/linux/blkdev.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/linux/blkdev.h b/include/linux/blkdev.h
index a97b1f29824f..33221dfc3581 100644
--- a/include/linux/blkdev.h
+++ b/include/linux/blkdev.h
@@ -58,7 +58,7 @@ struct blk_stat_callback;
  * Maximum number of blkcg policies allowed to be registered concurrently.
  * Defined here to simplify include dependency.
  */
-#define BLKCG_MAX_POLS		5
+#define BLKCG_MAX_POLS		7
 
 typedef void (rq_end_io_fn)(struct request *, blk_status_t);
 
-- 
2.20.0

